Update allow-members-to-send-as-or-send-on-behalf-of-group.md (#7927)
MicrosoftDocs/microsoft-365-docs: microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md

  
1. In the Exchange admin center, go to **Recipients** \> <a href="https://go.microsoft.com/fwlink/?linkid=2183233" target="_blank">**Groups**</a>.
    

> 2. Select the Group that you want to allow users to send as. 

Reviewer: ```suggestion
2. Select the group that you want to allow users to send as. 
```

> 3. Select **Settings** and Click on **Edit manage delegates**

Reviewer: ```suggestion
3. Select **Settings** > **Edit manage delegates**.
```

> 4. In the **Add a delegate** section, Type the email address of the user that you would like have send as acess.

Reviewer: ```suggestion
4. In the **Add a delegate** section, enter the email address of the user that you would like to have **Send as** access.
```

> 5. Select **Permission Type** as **Send as** from the drop down.

Reviewer: ```suggestion
5. Select **Permission Type** as **Send as** from the drop-down.
```

> 6.  Click on **Save changes**

Reviewer: ```suggestion
6.  Select **Save changes**.
```

    
    
## Allow members to send email on behalf of a group

This section explains how to allow users to send email on behalf of a group in the <a href="https://go.microsoft.com/fwlink/p/?linkid=2059104" target="_blank">Exchange admin center (EAC)</a> in Exchange Online.
  
1. In the Exchange admin center, go to **Recipients** \> <a href="https://go.microsoft.com/fwlink/?linkid=2183233" target="_blank">**Groups**</a>.
    

> 2. Select the Group that you want to allow users to send on behalf. 

Reviewer: ```suggestion
2. Select the group that you want to allow users to send on behalf of. 
```

> 3. Select **Settings** and Click on **Edit manage delegates**

Reviewer: ```suggestion
3. Select **Settings** > **Edit manage delegates**.
```

> 4. In the **Add a delegate** section, Type the email address of the user that you would like have send as acess.

Reviewer: ```suggestion
4. In the **Add a delegate** section, enter the email address of the user that you would like to have **Send as** access.
```

> 5. Select **Permission Type** as **Send on behalf** from the drop down.

Reviewer: ```suggestion
5. Select **Permission Type** as **Send on behalf** from the drop-down.
```

> 6.  Click on **Save changes**

Reviewer: ```suggestion
6.  Select **Save changes**.
```
